src/app/core/models/issue.model.ts

```diff
@@ -113,6 +115,9 @@ export class Issue {
     this.duplicated = !!githubIssue.findLabel(GithubLabel.LABELS.duplicated, false);
     this.status = githubIssue.findLabel(GithubLabel.LABELS.status);
     this.pending = githubIssue.findLabel(GithubLabel.LABELS.pending);
+
+    this.teamChosenSeverity = null;
```

Comment:
I think we can remove these initialisers for now, for consistency (as we are not initialising other fields that are phase-specific, such as `duplicateOf`)

Reply:
Sure no problem